waku/persistence/store.go

```diff
@@ -84,6 +104,28 @@ func (d *DBStore) createTable() error {
 	return nil
 }
 
+func (d *DBStore) cleanOlderRecords() error {
```

Comment:
shall we have a goroutine cleaning the record periodically?

Reply:
Not sure, currently go-waku only fetches the records from the DB during login, and this function would have been executed before so the records returned would be valid according to the retention policy